src/components/collapsible/Collapsible.scss

```diff
@@ -29,6 +29,7 @@
     .collapsible-card-header-caret {
         position: absolute;
         right: 0;
+        transition: transform $bdl-transition-duration-200ms;
```

Comment:
I'd lean toward `-short` instead of a precise value in the variable name, or something similar. We can define a few specific speeds, and avoid future var.s like `-300ms`, `-450ms`, etc.

I'd also consider defining the easing function too. One here is using `ease-in` and the ones without will default to `linear`. (IMO `ease-in-out` offers the smoothest timing behavior). 

This makes me think a mixin for the transition might be more useful, so we can capture all three parts. Thoughts?

Reply:
I'll do it in a followup